pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -65,7 +65,13 @@ def deserialize(self, parser):
         except Exception:
             sections = parser._sections.keys()
 
-            for section in sections:
+            # ProductMD follows a specific order to deserialize:
+            # https://github.com/release-engineering/productmd/blob/master/productmd/treeinfo.py#L120
+            deserialize_order = "header release tree variants checksums images stage2 media".split()
```

Comment:
It's an interesting approach, it wouldn't cross my mind to create a list this way!

I'm on the fence. 
Having a list (or maybe a tuple in this case is better) explicitly seems to me the obvious choice because the goal is to have a sequence. At the same time it would mean many quotation marks which can be seen as bringing noise.
Having it as a string and then split is a bit unintuitive way to introduce a sequence but there is no noise with quotes.